Tasks/XamariniOS/xamarinios.ts

```diff
@@ -152,7 +101,7 @@ async function run() {
             buildRunner.line(args);
         }
         buildRunner.argIf(codesignKeychain, '/p:CodesignKeychain=' + codesignKeychain);
-        if (buildTool === 'msbuild' && signIdentity && signIdentity.indexOf(',') > 0) {
+        if (signIdentity && signIdentity.indexOf(',') > 0) {
```

Comment:
Are you sure that's true?  IIRC, (and it's very possible I don't RC) we use `indexOf` because of TS / JS version compatibility.  Do we support the version of Javascript in the tasks project that can use the `contains` method?

Reply:
Sorry this is supposed to be `includes`; I've updated my original comment.
`String.prototype.includes` was introduced with ES 2015.  I tested it on Node 5 and 6 just now and it is supported on both.

Reply:
👍